pkg/pget.go

```diff
@@ -70,6 +71,8 @@ func (g *Getter) DownloadFiles(ctx context.Context, manifest Manifest) (int64, t
 	}
 
 	errGroup, ctx := errgroup.WithContext(ctx)
+	// TODO: un-hardcode this
+	errGroup.SetLimit(20)
```

Comment:
I'd really like to make this at the very least a ENV configurable if not a hidden cli flag.